src/server/adminBro.ts

```diff
@@ -2005,8 +2006,15 @@ export const verifyProjects = async (
   // prioritize revokeBadge
   const verificationStatus = revokeBadge ? false : verified;
   try {
+    const updateParams = { verified: verificationStatus };
+
+    if (verificationStatus) {
+      const key = 'verificationStatus';
+      updateParams[key] = null;
+    }
```

Comment:
If the project is verified, the verification status for revoking is reset to null.